nesta/core/schemas/tier_1/remappings/health_scanner.json

```diff
@@ -0,0 +1,167 @@
+{
+    "type_of_entity": {
+        "crunchbase": "company",
+        "meetup": "meetup",
+        "nih": "project"
+    },
+    "id": {
+        "meetup": "id_of_group",
+        "nih": "id_of_project"
+    },
+    "name": {
+        "crunchbase": "name_parent_organisation",
+        "meetup": "name_of_group",
+        "nih": "name_of_project"
+    },
+    "description": {
+        "crunchbase": "textBody_descriptive_organisation",
+        "meetup": "textBody_descriptive_group",
+        "nih": "textBody_descriptive_project"
+    },
+    "brief": {
+        "crunchbase": "textBody_summary_organisation",
+        "meetup": "textBody_abstract_project"
+    },
+    "title": {
+        "nih": "title_of_project"
+    },
+    "start_date": {
+        "crunchbase": "date_birth_organisation",
+        "meetup": "date_start_group",
+        "nih": "date_start_project"
+    },
+    "end_date": {
+        "crunchbase": "date_death_organisation",
+        "nih": "date_end_project"
+    },
+    "update_date": {
+        "crunchbase": "datetime_updated_organisation"
+    },
+    "continent": {
```

Comment:
also I wanted to propose a `geo_*` namespace as we'd also have `funding_*`, `terms|array_*`, `url_*`